bin/pos-cli-data-clean.js

```diff
@@ -33,17 +33,14 @@ const confirmCleanup = async (gateway, inlineConfirmation) => {
     clean(gateway);
   } else {
     logger.Error('Wrong confirmation. Closed without cleaning instance data.');
-    process.exit(1);
```

Comment:
@pavelloz But why?

Reply:
https://github.com/mdyd-dev/pos-cli/blob/dd3e7fd539aa2a6c7f903a458067ef9b2c85f88a/lib/logger.js#L25

Why not?

Reply:
You guys make me lol sometimes.